src/app/landing/landing.component.html

```diff
@@ -119,7 +120,12 @@
         <span *ngFor="let vhr of visibleHeatMap | keyvalue"
           >{{ vhr.key | renameTargetType }} {{ vhr.value ? 2030 : 2025 }}</span
         >
-        <a class="x" title="remove target layer" (click)="clearHeatmap()"></a>
+        <a
+          class="x"
```

Comment:
my suggestion would be to place a name to this class `x`, I interpret like generic or  `closeCountry` or `closeTarget`. something to consider.

Reply:
@jeortizquan I think 'x' is the best name for an 'x' - it's descriptive!  It's generic (it doesn't have to be a button, and doesn't necessarily have to close).  We use it elsewhere, but I take your point - it's difficult to search the codebase for an 'x'!